bootcd/rpms/genesis_scripts/genesis_scripts.spec

```diff
@@ -49,10 +56,13 @@ install -m 555 -T %{SOURCE4}   $RPM_BUILD_ROOT/usr/bin/genesis-bootloader
 
 %files
 %defattr(-, root, root)
+%config /etc/init.d/genesis
 %config /etc/init.d/network-prep
 %config /etc/sysconfig/init.diff
+%config /etc/sysconfig/genesis
 %config /etc/init/tty.conf.override
 %config /root/.bash_profile.genesis_scripts
+%config /root/run-genesis-bootloader
```

Comment:
It's not really a config though... I'd remove this config directive to prevent odd behavior on upgrade when booted into live image (i.e. when testing)